pkg/validations/base.go

```diff
@@ -37,6 +38,22 @@ func RunValidations(request reconcile.Request, obj client.Object, kind string, i
 		return
 	}
 
+	// If controller has no replicas clear existing metrics and
+	// do not run any validations
+
+	objValue := reflect.Indirect(reflect.ValueOf(obj))
+	spec := objValue.FieldByName("Spec")
+	if spec.IsValid() {
+		replicas := spec.FieldByName("Replicas")
```

Comment:
Need to verify that replicas is valid.  Spec field can exist on an object w/o replicas existing in the spec